packages/edit-site/src/components/layout/style.scss

```diff
@@ -1,80 +1,102 @@
+$hub-height: $grid-unit-20 * 2 + $button-size;
+
 .edit-site-layout {
-	min-height: 100%;
+	height: 100%;
 	background: $gray-900;
 	color: $white;
 	display: flex;
 	flex-direction: column;
-	@include break-small {
-		display: grid;
-		grid-template-columns: 320px 1fr;
-		grid-template-rows: auto 1fr;
-		grid-template-areas:
-			"header canvas"
-			"sidebar canvas";
-
-		&.is-full-canvas {
-			grid-template-areas:
-				"header header"
-				"canvas canvas";
-		}
-	}
 }
 
-.edit-site-layout__header {
-	grid-area: header;
-	height: $header-height;
+.edit-site-layout__hub {
+	position: fixed;
+	top: $canvas-padding;
+	left: $canvas-padding;
+	width: calc(100vw - #{$canvas-padding * 2});
+	height: $hub-height;
+	z-index: 3;
```

Comment:
Should we remove this hardcoded value and create a z-index function value for this?